include/picongpu/plugins/radiation/amplitude.hpp

```diff
@@ -92,16 +93,28 @@ namespace picongpu
                 {
                 }
 
+                /** constructor
+                 *
+                 *  @param x
```

Comment:
please remove `param ...` or add some text.

Reply:
I added a description of the parameters as well as of the constructor instead.